libres_util/src/res_log.c

```diff
@@ -16,14 +16,23 @@
    for more details. 
 */
 
+/*TODO:
```

Comment:
Probably not the place for these maybe, but where else? At least it is close to the code, and that is kind of nice.